libvmi/driver/kvm/include/kvmi/libkvmi.h

```diff
@@ -102,10 +111,17 @@ int     kvmi_get_registers( void *dom, unsigned short vcpu, struct kvm_regs *reg
                             struct kvm_msrs *msrs, unsigned int *mode );
 int     kvmi_set_registers( void *dom, unsigned short vcpu, const struct kvm_regs *regs );
 int     kvmi_shutdown_guest( void *dom );
-int     kvmi_reply_event( void *dom, unsigned int msg_seq, const void *data, unsigned int data_size );
+int     kvmi_reply_event( void *dom, unsigned int msg_seq, const void *data, size_t data_size );
 int     kvmi_pop_event( void *dom, struct kvmi_dom_event **event );
-int     kvmi_wait_event( void *dom, int ms );
+int     kvmi_wait_event( void *dom, kvmi_timeout_t ms );
 void    kvmi_set_log_cb( kvmi_log_cb cb, void *ctx );
+void *  kvmi_batch_alloc( void *dom );
+int     kvmi_batch_commit( void *batch );
+void    kvmi_batch_free( void *batch );
+int     kvmi_queue_registers( void *batch, unsigned short vcpu, const struct kvm_regs *regs );
+int     kvmi_queue_reply_event( void *batch, unsigned int msg_seq, const void *data, size_t data_size );
+int     kvmi_queue_page_access( void *batch, unsigned long long int *gpa, unsigned char *access, unsigned short count );
+int     kvmi_queue_pause_vcpu( void *batch, unsigned short vcpu );
```

Comment:
@adlazar i believe there is a mismatch between the `libkvmi.h` from KVM and this one in the PR,
because the `kvmi_get_maximum_gf` API is missing here.
https://github.com/adlazar/kvm/blob/kvmi-v6/tools/kvm/kvmi/include/kvmi/libkvmi.h#L125

Reply:
I just ran a `diff` and it's the only API missing.

Reply:
The implementation is missing too.

Reply:
Commit c927ebb adds the new command.